easybuild/easyconfigs/h/HTSeq/HTSeq-0.6.1p1-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "PythonPackage"
+
+name = 'HTSeq'
+version = '0.6.1p1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www-huber.embl.de/users/anders/HTSeq/'
+description = """A framework to process and analyze data from high-throughput sequencing (HTS) assays"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[0:2])
+eggname = '%(name)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg'
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('matplotlib', '1.5.1', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': ["bin/htseq-count", "bin/htseq-qa"],
+    'dirs': ["lib/python%s/site-packages/%s" % (pyshortver, eggname)],
```

Comment:
@RvDijk please use `"lib/python%%(pyshortver)s/site-packages/%s" % eggname`

the double `%` escapes the `%(pyshortver)s` template initially